inference/nlp/realtime/byoc/multi_model_catboost.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<p>We are no longer uploading 100 copies, right?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3710/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
right only 5 copies

Reply:
<p>Can we change this text then? Otherwise customers may be confused about the number of models</p>

Reply:
updated in the latest push